components/ui/code-block/code-block.tsx

```
@@ -0,0 +1,204 @@
+  // Return early if code block is closed
+  if (!isVisible) {
+    return (
+      <div className="relative group overflow-hidden my-6 w-full flex justify-center">
+        <div className="relative max-w-full w-full shadow-md" style={{
+          overflow: 'hidden',
+          borderRadius: '8px'
+        }}>
+          <div className={cn(
+            "flex items-center bg-[#1a2a35] border border-gray-700/50 rounded-lg cursor-pointer",
+            "px-2 sm:px-3 md:px-4 py-0.5 sm:py-1 md:py-1.5" // Reduced height
+          )} onClick={handleClose}>
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)